src/test/java/redis/clients/jedis/ClusterPipeliningTest.java

```diff
@@ -0,0 +1,883 @@
+        sorted.add("4");
+        sorted.add("5");
+
+        ClusterConnectionProvider provider = new ClusterConnectionProvider(nodes, DEFAULT_CLIENT_CONFIG);
+        ClusterPipeline p = new ClusterPipeline(provider);
+
+        Response<Long> r1 = p.rpush("key1", "2","3","5","1","4");
+        Response<List<String>> r2 = p.sort("key1");
+        Response<Long> r3 = p.sort("key1", "key1");
+        Response<List<String>> r4 = p.lrange("key1", 0, 4);
+        Response<List<String>> r5 = p.sort("key1", new SortingParams().limit(0, 2));
+        Response<Long> r6 = p.sort("key1", new SortingParams().desc(), "key1");
+        Response<List<String>> r7 = p.lrange("key1", 0, 4);
+
+        p.sync();
+        Assert.assertEquals(Long.valueOf(5), r1.get());
+        Assert.assertEquals(sorted, r2.get());
+        Assert.assertEquals(Long.valueOf(5), r3.get());
+        Assert.assertEquals(sorted, r4.get());
+        Assert.assertEquals(2, r5.get().size());
+        Assert.assertEquals(Long.valueOf(5), r6.get());
+        Collections.reverse(sorted);
+        Assert.assertEquals(sorted, r7.get());
+    }
+
+    @Test
+    public void clusterPipelineList() {
+        List<String> vals = new ArrayList<>();
+        vals.add("foobar");
+
+        ClusterConnectionProvider provider = new ClusterConnectionProvider(nodes, DEFAULT_CLIENT_CONFIG);
+        ClusterPipeline p = new ClusterPipeline(provider);
+
+        Response<Long> r1 = p.lpush("mylist", "hello", "hello", "foo", "foo"); // ["foo", "foo", "hello", "hello"]
+        Response<Long> r2 = p.rpush("mynewlist{$}", "hello", "hello", "foo", "foo");  // ["hello", "hello", "foo", "foo"]
+        Response<Long> r3 = p.lpos("mylist", "foo");
+        Response<Long> r4 = p.lpos("mylist", "foo", new LPosParams().maxlen(1));
+        Response<List<Long>> r5 = p.lpos("mylist", "foo", new LPosParams().maxlen(1), 2);
+        Response<String> r6 = p.ltrim("mylist", 2, 3); // ["hello", "hello"]
+        Response<Long> r7 = p.llen("mylist");
+        Response<String> r8 = p.lindex("mylist", -1);
+        Response<String> r9 = p.lset("mylist", 1, "foobar"); // ["hello", "foobar"]
+        Response<Long> r10 = p.lrem("mylist", 1, "hello"); // ["foobar"]
+        Response<List<String>> r11 = p.lrange("mylist",0,10);
+        Response<String> r12 = p.rpop("mynewlist{$}"); // ["hello", "hello", "foo"]
+        Response<List<String>> r13 = p.lpop("mylist", 1); // ["foobar"]
+        Response<List<String>> r14 = p.rpop("mynewlist{$}", 2); // ["hello"]
+        Response<Long> r15 = p.linsert("mynewlist{$}", ListPosition.AFTER, "hello", "world"); // ["hello", "world"]
+        Response<Long> r16 = p.lpushx("myotherlist{$}", "foo", "bar");
+        Response<Long> r17 = p.rpushx("myotherlist{$}", "foo", "bar");
+        Response<String> r18 = p.rpoplpush("mynewlist{$}", "myotherlist{$}");
+        Response<String> r19 = p.lmove("mynewlist{$}", "myotherlist{$}", ListDirection.LEFT, ListDirection.RIGHT);
```

Comment:
Could you please use a better hashtag? E.g. `list`, as in `my{list}`, `mynew{list}`, etc.